src/rgw/driver/rados/rgw_notify.cc

```diff
@@ -391,6 +389,14 @@ class Manager : public DoutPrefixProvider {
 
       // delete all published entries from queue
       if (remove_entries) {
+        uint64_t entries_to_remove = 0;
```

Comment:
it is used in:
```
cls_2pc_queue_remove_entries(op, end_marker, entries_to_remove);
```
this will update the entries number in header of the queue